src/test/java/teammates/test/cases/ui/AllActionsAccessControlTest.java

```diff
@@ -73,7 +73,7 @@ private static void addUnregStudentToCourse1() throws Exception {
     }
 
     @Test
-    public void InstructorCoursesPage() throws Exception {
+    public void instructorCoursesPage() throws Exception {
```

Comment:
Yes it is, but that means it's either we change this to `test*`, or we change everything else (1000+ occasions) to `*` (no `test` prefix).